tests/models/test_match_spec.py

```diff
@@ -717,3 +717,60 @@ def test_parse_build_number_brackets(self):
         #     "build_number": '>=3',
         # }
 
+
+class MatchSpecMergeTests(TestCase):
+
+    def test_merge_single_name(self):
+        specs = (MatchSpec('exact'), MatchSpec('exact 1.2.3 1'), MatchSpec('exact >1.0,<2'))
+        merged_specs = MatchSpec.merge(specs)
+        print(merged_specs)
+        assert len(merged_specs) == 1
+        merged_spec = merged_specs[0]
+        print(merged_spec)
+        assert str(merged_spec) == "exact[version='1.2.3,>1.0,<2',build=1]"
+        assert merged_spec.match({
+            'name': 'exact',
+            'version': '1.2.3',
+            'build': '1',
+            'build_number': 1,
+        })
+        assert not merged_spec.match({
+            'name': 'exact',
+            'version': '1.2.2',
+            'build': '1',
+            'build_number': 1,
+        })
+
+    def test_merge_multiple_name(self):
+        specs = tuple(MatchSpec(s) for s in (
+            'exact', 'exact 1.2.3 1', 'exact >1.0,<2',
+            'bounded >=1.0,<2.0', 'bounded >=1.5', 'bounded <=1.8',
```

Comment:
How about intermixing the specs to make sure order doesn't matter? E.g.:
```python
            'exact', 'exact 1.2.3 1',
            'bounded >=1.0,<2.0', 'bounded >=1.5', 'bounded <=1.8',
            'exact >1.0,<2',
```

Reply:
done